optics-core/src/Data/Tuple/Optics.hs

```diff
@@ -1,47 +1,1280 @@
+-------------------------------------------------------------------------------
+-- |
+-- Module      :  Data.Tuple.Optics
+-- Copyright   :  (C) 2012-16 Edward Kmett
+-- License     :  BSD-style (see the file LICENSE)
+-- Maintainer  :  Edward Kmett <ekmett@gmail.com>
+-- Stability   :  provisional
+-- Portability :  Rank2Types
+--
+-------------------------------------------------------------------------------
 module Data.Tuple.Optics
-  ( _1
-  , _2
-  , curried
-  , uncurried
-  , swapped
```

Comment:
these three, `curried`, `uncurried` and `swapped` are moved to `Iso`, or are they gone?